package.json

```diff
@@ -4,7 +4,7 @@
     "access": "public"
   },
   "url": "https://github.com/material-table-core/core",
-  "version": "4.3.10",
+  "version": "4.3.11",
```

Comment:
Can you not increase the version since its connected to our release cycle?

Reply:
reverted ! sorry it's an old habit :smiley: